### ISSUE TITLE
Don't loop samples that aren't looped initially

### ISSUE THREAD

**Issue by villermen** [team-thyme/soundboard#53]
When playing multiple samples of the same time with <kbd>Shift</kbd>, and after that you press one with <kbd>Shift</kbd> and <kbd>Ctrl</kbd>, all of the previous ones (that haven't yet ended) will be looped. This is because loop is set on the howl property instead of the group.

Theoretically this would be fixed by setting the loop property like this: 

```
    const howlerId = sample.howl.play();
    sample.howl.loop(loop, howlerId);
```

This will not make the sample loop at all however. I wonder if it might be a bug in Howler.

**Comment by Mesoptier** [comment]
Huh, this shouldn't be an issue until we implement https://github.com/villermen/soundboard/issues/31.

**Comment by villermen** [comment]
Oh, about that: 8ebba7feb30b7853e7ca726f2e070f1531b15de2.

**Comment by Mesoptier** [comment]
Ja, kijk...

**Comment by villermen** [comment]
This was actually fixed in ee1e3c07fe8b65f6dfe83438560b7ae98e525dea.
